libraries/src/concentrated/SQ63x64.sw

```diff
@@ -142,85 +142,80 @@ impl core::ops::Subtract for SQ63x64 {
             value: self.value - other.value,
         }
     }
-}
+} 
 impl SQ63x64 {
     /// Multiply a SQ63x64 with a SQ63x64. Panics of overflow.
     //TODO: assumes positive values
-    fn multiply(self, other: Self) -> Self {
+    pub fn multiply(self, other: Self) -> Self {
```

Comment:
Isn't this inherently public because its in a impl block not on a trait?

Reply:
Believe you are correct yes. We'll know for sure as we test more.